tests/e2e/config/jest.config.js

```diff
@@ -0,0 +1,12 @@
+const path = require( 'path' );
+const { useE2EJestConfig } = require( '@woocommerce/e2e-environment' );
+
+// eslint-disable-next-line react-hooks/rules-of-hooks
+const config = useE2EJestConfig( {
+	moduleFileExtensions: [ 'js', 'ts' ],
+	roots: [ path.resolve( __dirname, '../specs' ) ],
+	testMatch: [ '**/*.(test|spec).(j|t)s', '*.(test|spec).(j|t)s' ],
+	testTimeout: 30000,
```

Comment:
:grey_question: 
Is that for a single test or all? 30mins look quite long, I'm worried it could block our travis.

Reply:
That is for all the e2e tests, 30000 milliseconds (see [Jest docs on `testTimeout`](https://jestjs.io/docs/configuration#testtimeout-number)). It was copied from https://github.com/woocommerce/woocommerce-admin/blob/130eb95bae34a8efd83143c52cc08864efc15131/tests/e2e/config/jest.config.js.